build_odin.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I agree that it is the more portable way to invoke `bash`

Reply:
Yes, this means that the script can still run on OpenBSD without requiring changes to the SHABANG.